README.md

```
@@ -466,12 +528,203 @@ const paymentService = new Logger({
 const orderService = new Logger({
   prefix: '[OrderService]'
 });
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)